webgme/webgme-v1.d.ts

```diff
@@ -0,0 +1,31 @@
+
+/**
+Things in this module are effectively deprecated.
+This was a serialization supported in version 1.
+Primarily used for export.
+*/
+declare module "webgme/v1" {
```

Comment:
"webgme/v1" doesn't seem to be an actual module. If these are deprecated you could just not include them.